pandas/tests/io/excel/test_readers.py

```diff
@@ -778,7 +782,7 @@ def test_excel_passes_na(self, read_ext):
     @pytest.mark.parametrize('arg', ['sheet', 'sheetname', 'parse_cols'])
     def test_unexpected_kwargs_raises(self, read_ext, arg):
         # gh-17964
-        excel = ExcelFile('test1' + read_ext)
+        excel = pd.ExcelFile('test1' + read_ext)
```

Comment:
do we normally need to close the file? use in a context manager?

Reply:
It looks like it depends on what type of object ExcelFile is interacting with but ultimately converted usage to a context manager to be safe